### ISSUE TITLE
Bug sur le bouton "Installer notre outil JDMA"

### ISSUE THREAD

**Issue by AnthonyBrunelli** [DISIC/observatoire#1161]

**Comment by AnthonyBrunelli** [comment]
@lucaa tu me diras si ça prend du temps ?

**Comment by lucaa** [comment]
> @lucaa tu me diras si ça prend du temps ?

Non, ça à l'air d'être un problème d'escape lié à une modification de dernière minute, la #1153 . Je m'en occupe dans une petite quinzaine de minutes.

**Comment by lucaa** [comment]
Correctif fait: j'avais oublié des mises à jour de code dans le formulaire du bouton de la PA lors de l'implémentation de #1153. 

@AnthonyBrunelli c'est fait et installé en staging pour test.

**Comment by AnthonyBrunelli** [comment]
Vu, testé, tout est ok, merci @lucaa :)
